src/Form/IslandoraBasicCollectionAdmin.php

```diff
@@ -4,11 +4,33 @@
 
 use Drupal\Core\Form\ConfigFormBase;
 use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
  * Module administration form.
  */
-class IslandoraBasicCollectionAdmin extends ConfigFormBase {
+class IslandoraBasicCollectionAdmin extends ConfigFormBase implements ContainerInjectionInterface {
```

Comment:
Not sure if implementing `ContainerInjectionInterface` is needed? Similarly, is this form a good time to extend https://github.com/discoverygarden/islandora/blob/8.x/src/Form/IslandoraModuleHandlerAdminForm.php?